src/js/services/optimizely.js

```diff
@@ -43,6 +43,7 @@ class OptimizelyClient {
       }
     });
   }
+
```

Comment:
```suggestion
```

Reply:
I actually prefer it better with the space. Usually there is a space between comments and the function above. Ill change it back since I am not changing code in this area just why I added it